### PR TITLE
RPDs can now place disposals pipes on dense non-wall turfs

### LINKED ISSUE #16855: RPD's can't place disposal pipes under normal walls.
PR #14809 claimed to fix RPD's placing pipes under indestructible walls, however it made RPD's not be able to place disposal pipes under ANY wall. Something not stated as goal in the PR.

This makes building disposal networks a pain.

### PR DESCRIPTION
**Pull request by ChangelingRain**
Closes #16855

### CODE REVIEW

**Comment by Jalleo** [comment]
Aoparently not a bug and its intended

**Comment by lzimann** [comment]
Placing pipes on walls should be intended imo. Piping shit is already annoying enough.

**Comment by CPTANT** [comment]
It wasn't intended, people changed their mind about it later. The debate at the time was the RPD being used to circumvent unbreakable walls on away missions. It was never about preventing them from being build under normal walls.

**Comment by YakumoChen** [comment]
I can't pipe around the supermatter if I have to decon the walls first

**Comment by ChangelingRain** [comment]
You have to decon them currently. This is unlikely to change.
_ATMOS_ pipes can already be placed on walls.

**Comment by lzimann** [comment]
Ah, blocking disposals is fine.

File: code/__DEFINES/is_helpers.dm [hunk 1]
```diff
@@ -9,12 +9,16 @@
 //Turfs
 #define isopenturf(A) (istype(A, /turf/open))
 
+#define isindestructiblefloor(A) (istype(A, /turf/open/indestructible))
```
Comment (lzimann) [inline code/__DEFINES/is_helpers.dm]: why adding checks that are not even being used, reeee

**Review by Cyberboss — APPROVED** [review]

**Comment by lzimann** [comment]
\>post something about not used checks being added
\>not answered and merged anyway 
Thanks!

**Pull request merged**